client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java

```diff
@@ -69,7 +71,6 @@ public void registerJob(JobShuffleContext context) {
       synchronized (RemoteShuffleMaster.class) {
         if (lifecycleManager == null) {
           // use first jobID as celeborn shared appId for all other flink jobs
```

Comment:
this line should be moved as well?

Reply:
Thanks @pan3793 , I have made some modifications to the code for doing best effort to keep original Flink JobId.